src/Tribe/Main.php

```diff
@@ -291,7 +291,7 @@ public function before_html_data_wrapper( $html ) {
 
 			$data_attributes = array(
 				'live_ajax'         => tribe_get_option( 'liveFiltersUpdate', true ) ? 1 : 0,
-				'datepicker_format' => tribe_get_option( 'datepickerFormat' ),
+				'datepickerFormat'  => tribe_get_option( 'datepickerFormat' ),
```

Comment:
So I think it was my Sublime Text getting crazy. It marks the HTML as invalid on the "colors" for some reason.

I will change to the other one.